assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -0,0 +1,44 @@
+.component-direct-debit-pop-up-form {
+  z-index: 7500;
```

Comment:
Why so large?

Reply:
I put this number to match the scale used in `payment progress` message. We can review this practice and do a refactor in the future.